lib/src/components/position_component.dart

```diff
@@ -108,6 +108,10 @@ abstract class PositionComponent extends BaseComponent {
     return point.clone()..rotate(angle, center: position);
   }
 
+  PositionComponent({Vector2 position, Vector2 size})
```

Comment:
Nit: Add some empty spaces between the curly braces and the parameters please

Reply:
@erickzanardo that violates our styleguide though

Reply:
Everywhere else in the codebase we don't do spacing for this, so let's keep it consistent.:)

Reply:
Sure, I probably confused the styleguides, we use like I said at work